service/history/queue/timer_queue_processor_base.go

```diff
@@ -154,8 +155,13 @@ func (t *timerQueueProcessorBase) Start() {
 
 	t.redispatcher.Start()
 
+	newPollTime := time.Time{}
+	if startJitter := t.options.MaxStartJitterInterval(); startJitter > 0 {
+		now := t.shard.GetTimeSource().Now()
+		newPollTime = now.Add(time.Duration(rand.Int63n(int64(startJitter))))
+	}
 	for _, queueCollections := range t.processingQueueCollections {
-		t.upsertPollTime(queueCollections.Level(), time.Time{})
```

Comment:
From the outcome perspective, is there any difference between `time.Time{}` and `time.Now()`?

Reply:
there's no difference between those two. Both make the timerGate fire as soon as possible.